From e318cb7dcca03be7c4d8e1c03b36ffce5b4261ca Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Fri, 6 Sep 2013 11:30:44 +0800
Subject: [PATCH] ENGR00278504-5 imx_v7_defconfig: enable
 CONFIG_LOCALVERSION_AUTO

Used for uniquely identify a kernel version.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
[shawn.guo: cherry-pick commit 299d71ff7241 from imx_3.10.y]
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 1883eb2..93c5b1b 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -1,4 +1,3 @@
-# CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_KERNEL_LZO=y
 CONFIG_SYSVIPC=y
 CONFIG_NO_HZ=y
-- 
1.8.0

